src/ImageSharp/Processing/Quantization/QuantizedFrame{TPixel}.cs

```diff
@@ -1,61 +0,0 @@
-﻿// Copyright (c) Six Labors and contributors.
-// Licensed under the Apache License, Version 2.0.
-
-using System;
-using SixLabors.ImageSharp.PixelFormats;
-
-namespace SixLabors.ImageSharp.Processing.Quantization
-{
-    /// <summary>
-    /// Represents a quantized image frame where the pixels indexed by a color palette.
-    /// </summary>
-    /// <typeparam name="TPixel">The pixel format.</typeparam>
-    public class QuantizedFrame<TPixel>
```

Comment:
I'd actually like to keep this as a separate entity if I can.

Reply:
Reverted